.env.example

```diff
@@ -168,6 +168,13 @@ PERIODICAL=stop_expired_deploys:60,remove_expired_locks:60,report_system_stats:6
 # JENKINS_USERNAME= # user id
 # JENKINS_API_KEY= # API Token from user / Configure page
 
+## Jenkins Status Checker Plugin
+# JENKINS_STATUS_CHECKER=true
+# JENKINS_URL=https://jenkins.yourcompany.com
+# JENKINS_USERNAME=someguy@yourcompany.com
+# JENKINS_API_KEY=somekey
+# JENKINS_STAGING_VIEW=/view/StagingStatus
```

Comment:
does not need to be staging ... maybe do `JENKINS_STATUS_CHECKER=/view/StagingStatus` to solve both with a single flag ?